coderd/database/models.go

```diff
@@ -412,6 +413,70 @@ func AllBuildReasonValues() []BuildReason {
 	}
 }
 
+type ConnectionAction string
+
+const (
+	ConnectionActionConnect    ConnectionAction = "connect"
+	ConnectionActionDisconnect ConnectionAction = "disconnect"
+	ConnectionActionOpen       ConnectionAction = "open"
+	ConnectionActionClose      ConnectionAction = "close"
```

Comment:
Where is close used?

Reply:
Just remove it for now, enums can be added very easily in SQL but they can't be removed

Reply:
The types were added before the actual implementation. The idea was to support tracking app "close" (essentially timeout based) but in the end, the squeeze wasn't worth the lift and was left out. Absolutely fine to remove! 👍🏻